<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="Fußball"/>
  </property>
  <property name="styleSheet">
   <string notr="true">backgounrd-color: rgb(84, 84, 84)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="grid"/>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="compare">
        <property name="text">
         <string>Vergleichen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="team2">
        <item>
         <property name="text">
          <string>Ägypten</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Ägypten.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Argentinien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Argentinien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Australien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Australien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Belgien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Belgien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brasilien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Brasilien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Costa Rica</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Costa Rica.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dänemark</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Dänemark.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deutschland</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Deutschland.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>England</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/England.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Frankreich</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Frankreich.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Iran</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Iran.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Island</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Island.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Japan</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Japan.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kolumbien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Kolumbien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kroatien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Kroatien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marokko</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Marokko.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mexiko</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Mexiko.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nigeria</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Nigeria.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Panama</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Panama.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Peru</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Peru.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polen</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Polen.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portugal</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Portugal.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russland</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Russland.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saudi-Arabien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Saudi-Arabien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Schweden</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Schweden.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Schweiz</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Schweiz.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Senegal</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Senegal.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serbien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Serbien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Spanien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Südkorea</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Südkorea.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tunesien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Tunesien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Uruguay</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Uruguay.png</normalon>
          </iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="team1">
        <item>
         <property name="text">
          <string>Ägypten</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Ägypten.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Argentinien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Argentinien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Australien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Australien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Belgien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Belgien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brasilien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Brasilien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Costa Rica</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Costa Rica.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dänemark</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Dänemark.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deutschland</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Deutschland.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>England</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/England.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Frankreich</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Frankreich.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Iran</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Iran.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Island</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Island.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Japan</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Japan.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kolumbien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Kolumbien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kroatien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Kroatien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marokko</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Marokko.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mexiko</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Mexiko.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nigeria</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Nigeria.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Panama</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Panama.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Peru</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Peru.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polen</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Polen.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portugal</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Portugal.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russland</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Russland.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saudi-Arabien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Saudi-Arabien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Schweden</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Schweden.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Schweiz</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Schweiz.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Senegal</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Senegal.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serbien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Serbien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Spanien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Südkorea</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Südkorea.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tunesien</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Tunesien.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Uruguay</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>../images/Uruguay.png</normalon>
          </iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="statistic1">
        <property name="text">
         <string>Zeige Statistik</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="statistic2">
        <property name="text">
         <string>Zeige Statistik</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
